### ISSUE TITLE
Game map should be bigger

### ISSUE THREAD

**Issue by janisozaur** [OpenRCT2/OpenRCT2#2782]
While watching a stream, it was suggested that we make map of park bigger or provide zoom levels, so it can be more functional.

**Comment by IntelOrca** [comment]
I assume you mean the map window right?

**Comment by janisozaur** [comment]
![zaznaczenie_091](https://cloud.githubusercontent.com/assets/550290/12465999/7ddaf89a-bfd1-11e5-8107-bcb913b51f22.png)

**Comment by Broxzier** [comment]
Some suggestions that I think would be nice to have:

- Showing the full map, now the corners are being chopped off
- Ability to drag the view with the left mouse button - perhaps remove the scrollbars entirely, and have the right mouse button handle to 'move to target' operation?
- Zooming in and out

**Comment by Gymnasiast** [comment]
@Broxzier:
* Showing the full map, now the corners are being chopped off
 * +1
* Ability to drag the view with the left mouse button - perhaps remove the scrollbars entirely, and have the right mouse button handle to 'move to target' operation?
 * That is almost exactly the opposite of how the left and right buttons behave in the game: right drags the view, left does other things. Having right drag the view is a better idea IMO. Also, the scrollbars are easily discoverable for new users, and thus should be kept, with dragging with right click serving as a shortcut for power users.
* Zooming in and out
 * +1

**Comment by marijnvdwerf** [comment]
What about rotating the map by 45 degrees?

**Comment by Gymnasiast** [comment]
That would make the map not match the view, which is diamond-shaped as well.

**Comment by IntelOrca** [comment]
OpenTTD mini-map is isometric

**Comment by jez9999** [comment]
Yeah, minimap should totally have zoom functionality.  I need a magnifying glass to see it right now.

**Comment by ALEEF02** [comment]
+1 for zoom.
